client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -134,6 +136,66 @@ export const useCommandsArrayWpcom = ( {
 		displayNotice( __( 'SSH/SFTP password reset and copied to clipboard.' ) );
 	};
 
+	const getEdgeCacheStatus = async ( siteId: number ) => {
+		const response = await wpcom.req.get( {
+			path: `/sites/${ siteId }/hosting/edge-cache/active`,
+			apiNamespace: 'wpcom/v2',
+		} );
+
+		return response;
+	};
+
+	const clearEdgeCache = async ( siteId: number ) => {
+		try {
+			const response = await getEdgeCacheStatus( siteId );
+
+			if ( response ) {
+				// If global cache is active, purge the cache
+				await wpcom.req.post( {
+					path: `/sites/${ siteId }/hosting/edge-cache/purge`,
+					apiNamespace: 'wpcom/v2',
+				} );
+				displayNotice( __( 'Succesfully cleared cache.' ) );
```

Comment:
Also another heads-up: this notice is inside the condition because `clearWordPressCache` has its own custom notice so this is to prevent the duplication of success notices